pyanaconda/modules/timezone/timezone.py

```diff
@@ -167,6 +175,26 @@ def set_timezone_with_priority(self, timezone, priority):
         self.timezone_changed.emit()
         log.debug("Timezone is set to %s.", timezone)
 
+    def get_timezones(self):
+        """Get all valid timezones.
+
+        :return: list of valid timezones
+        :rtype: list of str
+        """
+        task = GetTimezonesTask()
+        timezone_dict = task.run()
```

Comment:
You don't need the task, this does not take time. Just call `get_all_regions_and_timezones()` here.